test/fixtures/organization_admin_user.js

```diff
@@ -8,5 +8,11 @@ module.exports = {
   // hash of 'youpin_admin' password
   password: '$2a$10$iorOMFOPboPeF20W20DKruey2UXXa4eOQSuReOMlxXnqNe5t6Egaq',
   email: 'organization_admin@youpin.city',
+  organization_and_role_pairs: [
+    { organization: '579334c74443625d6281b6dd', role: ORGANIZATION_ADMIN },
+  ],
+  organization_and_department_pairs: [
```

Comment:
Depends on organization policy? IMO, for the faculty of engineering, the responsibility of the Saraban team (organization admin) is screening and assigning tasks to other departments.